master/buildbot/worker/docker.py

```diff
@@ -79,6 +79,7 @@ def __init__(self, name, password, docker_host, image=None, command=None,
             except ValueError:
                 config.error("Invalid volume definition for docker "
                              "%s. Skipping..." % volume_string)
+                break
```

Comment:
Perhaps `continue` will be better?
User will be able to see all incorrectly formatted volumes.